TACC/Core-Styles
# hotfix: consistent margin-bottom for aligned images

## Overview

The left- and right-aligned images had a margin-bottom that a center-aligned image had not.

_@wesleyboar found no reason for this._

## Related

- required by TACC/Core-CMS#759

## Changes

- **added** missing selector to ruleset that adds margin-bottom

## Testing

1. Open http://localhost:3000/components/detail/align.
2. Verify margin-bottom for all aligned elements is the same.

## UI

| right | left | center |
| - | - | - |
| ![right](https://github.com/TACC/Core-Styles/assets/62723358/b81d5de3-a0c1-421a-a771-94b568b2e1b4) | ![left](https://github.com/TACC/Core-Styles/assets/62723358/7be24a93-bf0f-47c5-95bf-726630844d9a) | ![center](https://github.com/TACC/Core-Styles/assets/62723358/bc292bce-31fe-44e4-9f9f-3249a235531b) |
